measures/ApplyUpgrade/tests/apply_upgrade_test.rb

```diff
@@ -451,9 +519,8 @@ def _test_heat_pump_backup(heat_pump_type, heat_pump_is_ducted, expected_values)
 
       puts "\theat_pump_type='#{heat_pump_type}', heat_pump_is_ducted='#{heat_pump_is_ducted}'..."
 
-      heat_pump_backup_type = measure.get_heat_pump_backup_type(heating_system, heat_pump_type, heat_pump_is_ducted)
-      actual_values = measure.get_heat_pump_backup_values(heating_system)
-      actual_values['heat_pump_backup_type'] = heat_pump_backup_type
+      measure.set_existing_system_as_heat_pump_backup(runner, measures, hpxml_bldg)
```

Comment:
Call this parent method instead. Then we can check BuildResidentialHPXML argument assignments.